tests/main/ubuntu-core-gadget-config-defaults/task.yaml

```diff
@@ -55,10 +59,10 @@ restore: |
         echo "This test needs test keys to be trusted"
         exit
     fi
-    echo "Undo the rsyslog disable"
-    systemctl unmask rsyslog.service || true
-    systemctl enable rsyslog.service || true
-    systemctl start rsyslog.service || true
+    echo "Undo the service disable"
```

Comment:
Would we have a "if [ "$SERVICE" = ssh ]; then"  just like we have below? One unmasks, the other removes the confi file?

Reply:
yes, thanks for the review